### PR TITLE
Add goatcounter to slow zones

### PR DESCRIPTION
**Pull request by mathcolo**
- Add goatcounter to slow zones page
- Factor goatcounter and some constants out

### CODE REVIEW

File: src/analytics.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { PRODUCTION } from './constants';
+
+declare global {
+  interface Window { goatcounter: any; }
+}
+
+export function goatcount(path=window.location.pathname + window.location.search) {
```
Comment (austinjpaul) [inline src/analytics.ts:7]: whoa you can do this? this is breaking my poor python head

Comment (mathcolo) [reply src/analytics.ts:7]: yep! they're evaluated at call time

File: src/App.js [hunk 2]
```diff
@@ -120,11 +112,7 @@ class App extends React.Component {
       progressBarKey: 0,
     };
 
-    if (window.location.hostname === PRODUCTION) {
-      window.goatcounter?.count?.({
-        path: props.location.pathname,
-      });
-    }
+    goatcount(props.location.pathname);
```
Comment (austinjpaul) [inline src/App.js:115]: can this use the default param also? since window.location.search should be empty? or no...

**Review by austinjpaul — APPROVED** [review]
LGTM, thanks!!

File: src/App.js (hunk 2 above)
Comment (mathcolo) [reply src/App.js]: Yep! adjusted. that also let me to a double counting bug, fixed by moving the calls around a bit

**Pull request merged**